pyiceberg/catalog/dynamodb.py

```diff
@@ -81,6 +82,10 @@
 ITEM = "Item"
 
 
+@deprecated(
+    deprecated_in="0.6.0",
+    removed_in="1.0.0",
+)
```

Comment:
Shall we add some `help_message=` here? Otherwise, the outcome message becomes
```
DeprecationWarning: Call to DynamoDbCatalog, deprecated in 0.6.0, will be removed in 1.0.0.None
```
where the `None` is printed out. We could also modify the `deprecated`'s implementation to remove the `None`

Reply:
What additional information should be added here for the deprecation method. Also, yeah the `deprecated` implementation implies that the `help_message` is optional and the warning goes against that behavior.